test/it/user-change-password.js

```diff
@@ -40,11 +40,16 @@ describe('User API Tests', () => {
       newPassword: { value: '1234Abcd' }
     };
 
-    let response = await createdUser.changePassword(changePasswordCredentials);
-    expect(response.provider.type).to.equal('OKTA');
+    // Need to wait 1 second here as that is the minimum time resolution of the 'passwordChanged' field
+    await utils.delay(1000);
+    await createdUser.changePassword(changePasswordCredentials);
+
+    // 3. Verify that password was updated
+    const updatedUser = await client.getUser(createdUser.id);
+    expect(updatedUser.passwordChanged).to.be.gt(createdUser.passwordChanged);
     // TODO - Authenticate the user with updated password (Authn APIs not supported in SDK yet)
```

Comment:
Yes, we can remove the TODO. I'm anyway verifying that the password was updated by checking the `passwordChanged` field.